connector/connect/README.md

```diff
@@ -21,6 +21,17 @@ user experience across all languages. Please follow the below guidelines:
 
 Python-specific development guidelines are located in [python/docs/source/development/testing.rst](https://github.com/apache/spark/blob/master/python/docs/source/development/testing.rst) that is published at [Development tab](https://spark.apache.org/docs/latest/api/python/development/index.html) in PySpark documentation.
 
+When adding new proto messages, the python proto code need to be regenerated. To do it, use `dev/connect-gen-protos.sh` script.
+It depends on
+```
+brew install bufbuild/buf/buf
+```
+and python dependencies from
+```
+pip install -r dev/requirements.txt
+```
+(specifically, install `black` and dependencies from "Spark Connect python proto generation plugin (optional)" section)
+
```

Comment:
I would've written it differently.

````suggestion
To generate the Python client code from the proto files,

First, make sure to have a Python environment with the installed dependencies:

```
pip install -r dev/requirements.txt
```

Install [buf](https://github.com/bufbuild/buf)

```
brew install bufbuild/buf/buf
```

Generate the Python files by running:

```
dev/connect-gen-protos.sh
```

````